files/fr/conflicting/web/api/document_object_model_656f0e51418b39c498011268be9b3a10/index.html

```diff
@@ -10,7 +10,7 @@
 translation_of_original: Web/Guide/API/DOM
 original_slug: Web/Guide/DOM
 ---
-<p>{{draft}}</p>
+<p></p>
```

Comment:
```suggestion

```

Reply:
I removed all of these in #3090, I'm happy to remove it on this branch too if that is needed.